### ISSUE TITLE
bud6

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#2815]
- > actin nucleation-promoting factor Bud6

**Comment by ValWood** [comment]
could probably improve. Is a direct activator of formin

Formins associate with other nucleators and nucleation-promoting factors (NPFs) to stimulate collaborative actin assembly, but the mechanisms regulating these interactions have been unclear. Yeast Bud6 has an established role as an NPF for the formin Bni1, but whether it also directly regulates the formin Bnr1 has remained enigmatic. In this paper, we analyzed NPF-impaired alleles of bud6 in a bni1Δ background and found that Bud6 stimulated Bnr1 activity in vivo. Furthermore, Bud6 bound directly to Bnr1, but its NPF effects were masked by a short regulatory sequence, suggesting that additional factors may be required for activation. We isolated a novel in vivo binding partner of Bud6, Yor304c-a/Bil1, which colocalized with Bud6 and functioned in the Bnr1 pathway for actin assembly. Purified Bil1 bound to the regulatory sequence in Bud6 and triggered NPF effects on Bnr1. These observations define a new mode of formin regulation, which has important implications for understanding NPF-nucleator pairs in diverse systems.
PMID: 23671312
